contracts/Versioned.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity 0.5.16;
+
+contract Versioned {
```

Comment:
Is this based on any common pattern?

Reply:
I'm not sure. I wouldn't be surprised if lots of contracts use this. It just allows the client to know what version of the contract its dealign with so it can treat different versions differently. Its not used for for now needs to be there so that it is available in future